### PR TITLE
Issue 16

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--netcentric--hlxsites.hlx.page/insights/2022/09/take-your-cx-strategy
- After: https://issue-16--netcentric--hlxsites.hlx.page/insights/2022/09/take-your-cx-strategy

### CODE REVIEW

File: blocks/related-blogs/related-blogs.css [hunk 1]
```diff
@@ -10,3 +10,128 @@
     color: #97f5f7;
     margin-bottom: 25px;
 }
+
+.related-list-item {
+    background-color: #fff;
```
Comment (alexcarol) [inline blocks/related-blogs/related-blogs.css:15]: Maybe use `var(--c-white)`

File: blocks/related-blogs/related-blogs.css [hunk 2]
```diff
@@ -10,3 +10,128 @@
...
+
+div.teaser {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+}
+
+h2.teaser-description {
+    font-size: 22px;
+    color: #2e308e;
```
Comment (alexcarol) [inline blocks/related-blogs/related-blogs.css:54]: Same here, make sure to use --c-dark-plum

File: blocks/related-blogs/related-blogs.css [hunk 3]
```diff
@@ -10,3 +10,128 @@
...
+    height: 1em;
+    width: 1em;
+    display: inline-block;
+}
+
+.icon-wrapper svg {
+    pointer-events: none;
+    height: 100%;
+    width: 100%;
+    will-change: transform;
```
Comment (alexcarol) [inline blocks/related-blogs/related-blogs.css:119]: Mozilla developer docs only recommend using will-change as a lasst resort to deal with existing performance problems, consider removing it (https://developer.mozilla.org/en-US/docs/Web/CSS/will-change).

File: blocks/related-blogs/related-blogs.js [hunk 4]
```diff
@@ -1,12 +1,116 @@
...
-  // should we have a default if no tag configured?
-  return config.tags;
+  // should we have a default if no tag is configured?
+  return config.tag;
 }
+
+async function enrichProfiles(rArticles) {
+  const response = await fetch('/profile-blog.json');
+  const json = await response.json();
+  // console.debug(JSON.stringify(json.data));
```
Comment (alexcarol) [inline blocks/related-blogs/related-blogs.js:11]: make sure to remove the commented code

File: blocks/related-blogs/related-blogs.js [hunk 5]
```diff
@@ -1,12 +1,116 @@
...
+
+  Object.entries(rArticles).forEach((entry) => {
+    // eslint-disable-next-line no-unused-vars
+    const [key, value] = entry;
+    // eslint-disable-next-line no-restricted-syntax
```
Comment (alexcarol) [inline blocks/related-blogs/related-blogs.js:16]: Why is this for not allowed? If it's not useful, can we just disable that linter setting?

File: blocks/related-blogs/related-blogs.css [hunk 6]
```diff
@@ -10,3 +10,128 @@
     color: #97f5f7;
     margin-bottom: 25px;
 }
+
+.related-list-item {
```
Comment (mhaack) [inline blocks/related-blogs/related-blogs.css:14]: All the CSS definitions should be scoped with the block `.related-blogs`

File: blocks/related-blogs/related-blogs.js (hunk 4 above)
Comment (dkuntze) [reply blocks/related-blogs/related-blogs.js]: done

File: blocks/related-blogs/related-blogs.css (hunk 1 above)
Comment (dkuntze) [reply blocks/related-blogs/related-blogs.css]: done

File: blocks/related-blogs/related-blogs.css (hunk 6 above)
Comment (dkuntze) [reply blocks/related-blogs/related-blogs.css]: done

File: blocks/related-blogs/related-blogs.js (hunk 5 above)
Comment (dkuntze) [reply blocks/related-blogs/related-blogs.js]: fixed / removed

**Review by alexcarol — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Review by BiggsBottor — APPROVED** [review]
Looks good to me

**Pull request merged**